lib/raml-validator-loader.js

```diff
@@ -43,7 +43,7 @@ module.exports =
 /************************************************************************/
 /******/ ([
 /* 0 */
-/***/ function(module, exports, __webpack_require__) {
+/***/ (function(module, exports, __webpack_require__) {
```

Comment:
if I'm not mistaken this file is the `dist` generated after running `npm run build` in this repo.. can you confirm running that command please